recipes/snntorch/LICENSE

```diff
@@ -0,0 +1,33 @@
+GNU GENERAL PUBLIC LICENSE
```

Comment:
Since the license is already included in the PyPI tarball we're using, you don't need to manually include the file here. Please go ahead and remove (so we stay in sync with what the upstream package ships).